packages/xarc-react-router/src/common/index.ts

```diff
@@ -11,9 +11,10 @@ export type ReactRouterFeatureOptions = {
   /**
    * The browser history object - custom browser history object and control which Router to use.
    *
-   * - If it's falsy, then `BrowserRouter` will be used.
-   * - If it's `true`, then `Router` is used with history from `createBrowserHistory` from https://www.npmjs.com/package/history
-   * - Otherwise it's assumed to be a history object and `Router` will be used with it.
+   * - If it's false, then `BrowserRouter` will be used and it will use its own history.
+   * - Note: BrowserRouter use its own history object and its not shared among other subapps
+   * - If it's `true`, then `Router` is used with history from `createBrowserHistory` from https://www.npmjs.com/package/history and the same history object will be shared among all subapps
```

Comment:
what if `history` is not specified?

Reply:
I mentioned this here in the last line
* - Otherwise it's assumed to be a history object and `Router` will be used with it. If its undefined then share history object will be used for all subapps